internal/serverinstall/k8s.go

```diff
@@ -335,7 +290,7 @@ func (i *K8sInstaller) Install(
 		// since pods can't reach this.
 		if i.config.advertiseInternal || strings.HasPrefix(grpcAddr, "localhost:") {
 			advertiseAddr.Addr = fmt.Sprintf("%s:%d",
-				serviceName,
+				"waypoint-server",
```

Comment:
Can we keep this `serviceName`? Seems like it would be good to have some `const` in this file for repeat config values in the case where we wanna change something, we can change it once.